epgp.lua

```diff
@@ -456,6 +456,13 @@ local function ParseGuildNote(callback, name, note)
   GS:SetOutsidersEnabled(EPGP.db.profile.outsiders == 1)
 end
 
+function EPGP:IsRL()
```

Comment:
You may have seen this warning: "Make sure you are the only person changing EP and GP. If you have multiple people changing EP and GP at the same time, for example one awarding EP and another crediting GP, you **are** going to have data loss."

`EPGP:IsRLorML()` ensure that only one person will change EP and GP. Please use that function and remove `EPGP:IsRL()`.

Reply:
Will remove function and use IsRLorML